### PR TITLE
1154 support ref locids

### LINKED ISSUE #1154: Integrate loc/ids via specroutes gem

### PR DESCRIPTION
**Pull request by 0robustus1**
This branch/Pull-Request is based on 1154-add_loc-ids/#1179 instead of on staging.
The reason for this is the fact that it needs the previous loc/id work. As soon as the other pull-request
is merged, i will rebase on staging.

This Pull-Request completes the second half and shall therefore close #1154.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,29 +1,97 @@
...
+  specified_get '/ref/mmt/:repository_id/*path' => 'ontologies#show',
+    as: :ontology_iri_mmt,
+    constraints: [
+      MMTRouterConstraint.new(Ontology, ontology: :id),
+      MIMERouterConstraint.new('text/plain', 'text/html')
+    ]
+
+  specified_get '/ref/mmt/:repository_id/*path' => 'mappings#show',
+    as: :ontology_iri_mmt,
+    constraints: [
```
Comment (0robustus1) [inline config/routes.rb]: Again: No.

File: app/helpers/mappings_helper.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+module MappingsHelper
+  def sort_mapping_list(collection)
```
Comment (0robustus1) [inline app/helpers/mappings_helper.rb]: I've done all i could...

**Comment by eugenk** [comment]
test this build

**Comment by eugenk** [comment]
:+1:

**Pull request merged**